tests/unit/runner/test_service.py

```diff
@@ -45,11 +54,7 @@ async def test_service_entrypoint(mock_work_context, entrypoint, expected_script
     while s:
         scripts.append(s)
         future_results = asyncio.get_event_loop().create_future()
-
-        # the mock is sent as the return value of the `yield script` in the service
-        mock_exescript_awaitable = asyncio.sleep(0.01)  # type: ignore [var-annotated]
-
-        future_results.set_result(mock_exescript_awaitable)
+        future_results.set_result("some output...")
```

Comment:
I don't understand why do we change this (but changed version is probably also OK).

Reply:
yeah, I realized we _don't_ need to have an awaitable there - because the await works on the _future_ and not on the value